src/ViewLayer/DebugDisplay/AuxBMSPage/AuxBMSUi/AuxBMSUi.cpp

```diff
@@ -0,0 +1,125 @@
+#include "AuxBMSUi.h"
```

Comment:
The casing for this file name is inconsistent with the other files. It should probably be `AuxBmsUi`. Same goes for the folder and variable names .